spec.emu

```diff
@@ -168,15 +175,14 @@ contributors: Ujjwal Sharma, Younies Mahmoud
           1. Let _nf_ be ? Construct(%NumberFormat%, &laquo; _durationFormat_.[[Locale]], _nfOpts_ &raquo;).
           1. Let _num_ be ! FormatNumeric(_durationFormat_.[[NumberFormat]], _value_).
           1. Let _dataLocale_ be _durationFormat_.[[DataLocale]].
-          1. Let _localeData_ be %DurationFormat%.[[LocaleData]].
-          1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
+          1. Let _dataLocaleData_ be %DurationFormat%.[[LocaleData]].[[&lt;_dataLocale_&gt;]].
           1. If _style_ is `"2-digit"` or `"numeric"`, then
             1. Append the new Record { [[Type]]: _unit_, [[Value]]: _num_} to the end of _result_.
             1. If _unit_ is `"hours"` or `"minutes"`, then
               1. Let _separator_ be _dataLocaleData_.[[formats]].[[digital]].[[separator]].
               1. Append the new Record { [[Type]]: `"literal"`, [[Value]]: _separator_} to the end of _result_.
           1. Else,
-            1. Let _pr_ be ? Construct(%PluralRules&, &laquo; _durationFormat_.[[Locale]] &raquo;).
+            1. Let _pr_ be ? Construct(%PluralRules%, &laquo; _durationFormat_.[[Locale]] &raquo;).
```

Comment:
I was surprised to see simple syntactic errors like this. Does ecmarkup not complain about this?

Reply:
Surprisingly, it doesn't. Sorry errors like this crept in and thanks for fixing.